journal/202104.md

```diff
@@ -0,0 +1,351 @@
+# {DRAFT} Decred Journal – April 2021
+
+![{alt text}](../img/{file.ext} "{hover tooltip text}")
+
+_Image: {"$title by $author. $optional-description" for artwork, "$description" for photos}_
+
+April's highlights: {
+
+- pi v1.0 and proposal
+- Decrediton DEX integration progress
+- consensus vote/upgrade status
+
+}
+
+Contents: { sync }
+
+- [Development](#development)
+- [People](#people)
+- [Governance](#governance)
+- [Network](#network)
+- [Integrations](#integrations)
+- [Outreach](#outreach)
+- [Events](#events)
+- [Media](#media)
+- [Community Discussions](#community-discussions)
+- [Markets](#markets)
+- [Relevant External](#relevant-external)
+
+## Upgrade reminder
+
+{}
+
+## Development
+
+The work reported below has the "merged to master" status unless noted otherwise. It means that the work is completed, reviewed, and integrated into the source code that advanced users can [build and run](https://medium.com/@artikozel/the-decred-node-back-to-the-source-part-one-27d4576e7e1c), but is not yet available in release binaries for regular users.
+
+<a id="dcrd" />
+
+**[dcrd](https://github.com/decred/dcrd)**
+
+- codebase [converted](https://github.com/decred/dcrd/pull/2625) to use the new `stdaddr` package for handling addresses. As common for large changes, to make review easier this one was split into a series of individual commits such that everything builds and passes all tests each step of the way. { "make use of the script version" - how is this different from not making use of it previously? }
+- introduced [UTXO database](https://github.com/decred/dcrd/pull/2632) that will allow to optimize reading and writing of the UTXO set, and also decouple it from the other databases. { my text is blurry on why this is exciting (esp. the decoupling), better wording is welcome }
```

Comment:
It paves the way for a specialized UTXO database implementation that is tailored to the specific requirements of the UTXO set and therefore can be made more efficient in terms of both processing time and memory usage.